builder/amazon/chroot/step_mount_device.go

```diff
@@ -23,7 +24,8 @@ type mountPathData struct {
 //   mount_path string - The location where the volume was mounted.
 //   mount_device_cleanup CleanupFunc - To perform early cleanup
 type StepMountDevice struct {
-	mountPath string
+	mountPath    string
+	MountOptions []string
```

Comment:
Can you put this above `mountPath`? I like to put exported above unexported, example:

```
MountOptions []string

mountPath string
```